packages/authorization/README.md

```diff
@@ -1,83 +1,13 @@
 # @loopback/authorization
 
-A LoopBack 4 component for authorization support.
+A LoopBack 4 component for authorization support (Role based, Permission based,
```

Comment:
Concerning the paragraph using the word 'BasicStrategy':

![image](https://user-images.githubusercontent.com/6864736/70179641-643b1100-16ac-11ea-9469-a4d16bb1aacc.png)

Reply:
e.g.

![image](https://user-images.githubusercontent.com/6864736/70179760-a49a8f00-16ac-11ea-86e4-eade47ab26be.png)

Reply:
agreed , your original comment was resolved. This is a different MD file, I will fix this too.